src/Stratis.Bitcoin.Features.PoA/Voting/VotingData.cs

```diff
@@ -76,7 +76,14 @@ public override bool Equals(object obj)
         /// <inheritdoc />
         public override int GetHashCode()
         {
-            return this.Data.GetHashCode() ^ this.key;
+            ulong hash = 17;
```

Comment:
Why this change @quantumagi ?

Reply:
As far as I can remember the `GetHashCode` is based on object reference while we're instead interested in the content matching. Will refresh my memory on this.

Reply:
This wont have any effect on existing saved polls or anything like that? I suppose it shouldnt but just double checking.